docs/rfc/0698-cli-infrastructure.adoc

```diff
@@ -86,20 +86,51 @@ options, dubbed as "global parameters" here, will be a defined set --
 exported from `radicle-link` for re-use -- that the base `rad` command
 will always receive and pass onto the subcommand being called.
 
+These parameters will be prefixed by `rad-`, signfiying that they are
+from the `radicle-link` namespace and are reserved names.
+
+Subcommands, including the <<Core>> subcommands, MUST accept these
```

Comment:
Given the exception it might be better to make this SHOULD?

Reply:
You're dead right =] Fixed, thanks!